Classes/General.cs

```diff
@@ -34,6 +34,7 @@ public static class General {
             const int readBufferSize = 8192;
             const int requestTimeoutMS = 60000;
             const int readTimeoutMS = 60000;
+            string metaRedirectHtml;
```

Comment:
Declaration of this variable can be joined to its assignment in the inner scope.